scripts/quests/jeuno/Path_of_the_Bard.lua

```diff
@@ -0,0 +1,50 @@
+-- -----------------------------------
+-- -- Path of the Bard
+-- -----------------------------------
+-- -- Log ID: 3, Quest ID: 20
+-- -- Song Runes: !pos -717.633 -7.574 102.987 103
+-- -----------------------------------
+local ID = zones[xi.zone.VALKURM_DUNES]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.JEUNO, xi.quest.id.jeuno.PATH_OF_THE_BARD)
+
+quest.reward =
+{
+    gil      = 3000,
+    fame     = 30,
+    fameArea = xi.fameArea.JEUNO,
+    title    = xi.title.WANDERING_MINSTREL,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.questLog.JEUNO, xi.quest.id.jeuno.A_MINSTREL_IN_DESPAIR)
+        end,
+
```

Comment:
Eden files show missing dialogs found for this quest:
```lua
        [xi.zone.LOWER_JEUNO] = {
            ['Mataligeat'] = quest:event(144),
            ['Bki_Tbujhja'] = quest:event(182),
            ['Mertaire'] = quest:messageSpecial(JeunoID.text.COULD_HE_BE):importantOnce()
        },
```

Reply:
What does Eden have as Mataligeat's default response outside of the quests? I have 144 on LSB as his current default.

Reply:
shows 144 on default, so can omit that and just use the after quest to 143

Reply:
I left it out, but added in the other two responses